django-stubs/db/models/fields/__init__.pyi

```diff
@@ -120,6 +120,7 @@ class Field(RegisterLookupMixin, Generic[_ST, _GT]):
     remote_field: ForeignObjectRel | None
     is_relation: bool
     related_model: type[Model] | Literal["self"] | None
+    generated: bool
```

Comment:
```suggestion
    generated: ClassVar[bool]
```

See https://github.com/django/django/blob/1a36dce9c5e0627d46582829d7abd47ed872e3aa/django/db/models/fields/__init__.py#L174

Reply:
I'm having difficulty understanding this section. Could you please clarify why you opted to use `ClassVar`?

Reply:
Yes, most values defined on a class level are class var (unless their are overriden on an instance level). We need to keep this in mind :)

Reply:
Thank you for the clarification! I'll be sure to incorporate that into my future work as well!